app/api/helpers/custom_fields.py

```diff
@@ -245,3 +246,15 @@ def validate(self, value):
             self.validation_error = 'Value of %s is not in available choices'
             return False
         return True
+
+
+class LicenceClass(CustomField):
```

Comment:
It would be better if naming convention is followed here. Eg > All other classes in that file have names like String, Color, Boolean .. so I think you can make the class name as just 'Licence'. What do you think?